update.py

```diff
@@ -144,25 +144,379 @@ def commit_changes(githubtoken):
 
 
 parser = ArgumentParser()
-parser.add_argument('commands', nargs="+", help='Commands (starcount, commit, help)')
+parser.add_argument('commands', nargs="+", help='Commands (starcount, releases, releases_one, releases_asset, header, dates, commit, help)')
 parser.add_argument("--githubtoken", dest="githubtoken", help="Authentication token for GitHub API and ")
+parser.add_argument("--asset", dest="asset", help="Asset id (JSON file name without .json) to limit release update")
+parser.add_argument("--limit", dest="limit", type=int, help="Limit number of releases to fetch (default depends on command)")
 args = parser.parse_args()
 
 help = """
 COMMANDS:
 starcount = Add GitHub star count to all assets that have a GitHub project (requires --githubtoken)
+releases = Add sorted releases array (zip, tag, message). Use --asset=<id> to limit to one asset. Use --limit=N to cap result.
+releases_one = Add only the latest release (zip, tag, message) to assets. Combine with --asset=<id> for a single asset.
+releases_asset = Add releases only for one asset specified via --asset=<id> (requires --asset)
+header = Update or initialize header.json with timestamps for changed asset JSON files (or initialize all if missing)
```

Comment:
New commands. We need one request per-asset, It shouldn't be a problem for limits for now